jest/tests/case11/case11.index.js

```diff
@@ -0,0 +1,5 @@
+function foobar() {
```

Comment:
Do we need this file to test it? I think we just need the `.test.js` and 2 config files?

Reply:
@jbedard I followed other test cases, but you're right, we only need the `.test.js` files

Reply:
@jbedard applied in [`57d1180` (#127)](https://github.com/aspect-build/rules_jest/pull/127/commits/57d1180be66517ecc7c0be3e511134bfeeddc5e3)

Reply:
👍